crates/e2e/macro/src/config.rs

```diff
@@ -28,6 +28,8 @@ pub struct E2EConfig {
     whitelisted_attributes: WhitelistedAttributes,
     /// Additional contracts that have to be built before executing the test.
     additional_contracts: Vec<String>,
+    /// Custom environment for the contracts, if specified. Otherwise `DefaultEnvironment` is used.
```

Comment:
Can you link to the `Environment` and `DefaultEnvironment` docs here?

Reply:
:white_check_mark: